ci/ray_ci/doc/cmd_update_cache_env.py

```diff
@@ -17,6 +17,9 @@ def list_pending_files(ray_dir: str) -> List[str]:
         pending_files = f.readlines()
         pending_files = [file.strip() for file in pending_files]
     os.remove(f"{ray_dir}/{PENDING_FILES_PATH}")
+    for i in range(len(pending_files)):
+        if pending_files[i].split(".")[-1] != "py":
```

Comment:
add a private function `def _is_python_file(filename: str)` and reuse it here and line 85